targets/x11-common

```diff
@@ -19,16 +19,21 @@ if [ -n "$offset" ]; then
 fi
 
 # Install utilities and links for powerd-poking daemon
-compile xi2event '-lX11 -lXi' libx11-dev libxi-dev
-install --minimal dbus xdg-utils
+if [ "${DISTROAKA:-"$DISTRO"}" = 'gentoo' ]; then
+    compile xi2event '-lX11 -lXi' x11-libs/libXi
```

Comment:
I think the dependencies list is a crouton package selector, so you could combine these into `gentoo=x11-libs/libXi,libxi-dev gentoo=,libx11-dev`